### DISCUSSION TITLE
Is Edge support production-ready?

### DISCUSSION THREAD

**Discussion by luzdealba** [Azure/static-web-apps#1286] [General]
Every time we enable edge on our site, after a few days the site goes down.  And it took quite some back-and-forth to finally figure out that it was in fact enabling edge what was breaking the production deployment, since Azure support were initially clueless, and very open about how unfamiliar they were with Edge (which I actually appreciated).

We actually moved our operation to Static Web Apps (SWA) to make use of edge deployments; but now I see that on the SWA repo there are [quite a bunch of edge-related open issues](https://github.com/Azure/static-web-apps/issues?q=edge).

So, I gotta ask&mdash;**Is this so-called "enterprise-grade" edge production-ready?**

Does it even work?

**Comment by thomasgauvin** [comment]
Enterprise-Grade edge is GA and production ready! Can you share the support issue that is associated to the issue you encountered with the site going down? Would like to investigate what that is and if there are other factors at play

**Reply by luzdealba** [reply]
Hey Thomas, thanks for the comment.

Sure thing!  It's Service Request **#2309210030003987**.

We deploy using the SWA CLI instead of using a GitHub Action (not sure if that might play a role), and enabling edge seems to work at the beginning; but then after ~48 hours of propagation&mdash;apparently&mdash;production goes down with a 404, and immediately goes up again after disabling edge.

**Comment by jwhazel** [comment]
Was there ever any resolution to this? We've now observed this exact same behavior across two different sites on two separate occasions. The symptoms are identical - enable edge and 48 hours later it breaks. Disable edge and it immediately goes back to working.

I have a ticket open, and the support engineer has been nothing but pleasant, but we're going on a week and still don't have an answer why this keeps happening.

**Reply by luzdealba** [reply]
Hey Jesse,

After several days of troubleshoot, we managed to make the Edge setting stable by changing the DNS setup from A record to a CNAME record pointing to our Azure provisioned domain (some `xxx.number.azurestaticapps.net`).

It's been working okay since then--hope that helps!

**Reply by jwhazel** [reply]
Thank you so much for the quick reply, it was a huge help in finally tracking this down. After reading your message, I finally found something in the documentation that corroborates it: https://learn.microsoft.com/en-us/azure/static-web-apps/custom-domain
> Using A records is not recommended as your application will no longer benefit from its global distribution, and this may affect your application performance if your traffic is globally distributed.

If anyone at MS ever reads this, maybe consider making that verbiage more clear and also mentioning in the Edge docs: https://learn.microsoft.com/en-us/azure/static-web-apps/enterprise-edge?tabs=azure-portal
